### DISCUSSION TITLE
Publish URL for UI via MQTT

### DISCUSSION THREAD

**Discussion by hirsts** [arendst/Tasmota#12850] [Ideas]
It would be great if a full URI could be published via MQTT. This would result as an entity in Home Assistant that would provide a link to the specific Tasmota devices UI. 

I have raised a feature request in Home Assistant to make any publish URI clickable.

**Comment by sfromis** [comment]
Well, both host name and IP address are already published when connecting to MQTT. You could easily make a boot time rule to include such fields in a published payload, if you "need" to include the fixed prefix making either into an URL.
